script/setup

```diff
@@ -17,7 +17,7 @@ fi
 mkdir -p config
 
 if [ ! -n "$DEVCONTAINER" ];then
-  python3 -m venv venv
+  python3.8 -m venv venv
```

Comment:
Please don't change unrelated things.

Reply:
oops, sorry I didn't mean to push that. Removing now